daemon-tests/src/mocks/monitor.rs

```diff
@@ -37,7 +37,7 @@ impl MonitorActor {
 
     async fn handle(&mut self, _: monitor::StartMonitoring) {}
 
-    async fn handle(&mut self, _: monitor::CollaborativeSettlement) {}
+    async fn handle(&mut self, _: monitor::MonitorCollaborativeSettlement) {}
```

Comment:
🙃 I like the fact that we're being consistent and also that we're no longer reusing the name of the module here to give the correct meaning to this message. But `monitor::Monitor` is still a bit of an eyesore. What about changing the verb to `Watch` for all these messages that are currently prepended with `Monitor`?

Reply:
More than happy to make such a change but I'd like to get the lint in first and then iterate on such ideas so that they are applied consistently!

Reply:
The whole public API of `monitor` is pretty garbage TBH.